src/mqtt_client/mqtt_client.rs

```diff
@@ -62,10 +62,20 @@ where
         broker: IpAddr,
         clock: Clock,
     ) -> Result<Self, minimq::Error<Stack::Error>> {
-        let mqtt = minimq::Minimq::new(broker, client_id, stack, clock)?;
-
-        let mut response_topic: String<128> = String::from(prefix);
-        response_topic.push_str("/log").unwrap();
+        let mut mqtt = minimq::Minimq::new(broker, client_id, stack, clock)?;
+
+        // Configure a will so that we can indicate whether or not we are connected.
+        let mut connection_topic: String<75> = String::from(prefix);
+        connection_topic.push_str("/connected").unwrap();
```

Comment:
Can we use `/alive` as the recommended path for that? With `connected` it seems to unclear to me what's connected where (SMA connection at the ADC?)